src/main/java/org/alfresco/repo/search/impl/querymodel/impl/db/DBResultSet.java

```diff
@@ -190,61 +199,13 @@ public boolean hasMore()
         return new DBResultSetRowIterator(this);
     }
     
-    private void prefetch(int n)

```

Comment:
It could be used to bulk fetch `NodeRef`s but I don't think it has ever worked as calls to related methods all return fixed values that do not allow it

https://github.com/Alfresco/alfresco-repository/blob/1c065e39eab457ed75e54ba1076723cca4c08f0a/src/main/java/org/alfresco/repo/search/AbstractResultSet.java#L92-L126